### PR TITLE
Update gofmt and goimports

### PR DESCRIPTION
**Pull request by cdrage**
Updates imports as well as `gofmt`.

This is needed in order to reduce the amount of rebasing in new PR's
that use the auto-package-importer `goimports` as well as `gofmt`.

### CODE REVIEW

**Comment by cdrage** [comment]
This uses version 1.11.2 of Go:

```sh
github.com/redhat-developer/odo  satisfy-gofmt ✔                                                                                                                                                                                                                                                                                                                       1m  ⍉
▶ go version
go version go1.11.2 linux/amd64
```

For the formatting

**Comment by mik-dass** [comment]
@cdrage Maybe update travis.yml too?

**Comment by cdrage** [comment]
@mik-dass what do you mean?

**Comment by cdrage** [comment]
@mik-dass Ah, I see what you mean. I've gone ahead and updated the version to `1.11.2` within TravisCI.

File: pkg/component/watch.go [hunk 1]
```diff
@@ -2,14 +2,15 @@ package component
...
-	"github.com/redhat-developer/odo/pkg/util"
 	"io"
 	"os"
 	"path/filepath"
 	"regexp"
 	"sync"
 	"time"
 
+	"github.com/redhat-developer/odo/pkg/util"
+
```
Comment (kadel) [inline pkg/component/watch.go]: this empty line can be removed

File: pkg/odo/cli/catalog/catalog.go [hunk 2]
```diff
@@ -2,13 +2,14 @@ package catalog
...
 	"fmt"
-	"github.com/olekukonko/tablewriter"
-	"github.com/redhat-developer/odo/pkg/odo/genericclioptions"
-	"github.com/redhat-developer/odo/pkg/odo/util"
 	"os"
 	"strings"
 	"text/tabwriter"
 
+	"github.com/olekukonko/tablewriter"
+	"github.com/redhat-developer/odo/pkg/odo/genericclioptions"
```
Comment (kadel) [inline pkg/odo/cli/catalog/catalog.go]: we should have separate "groups" for odo pkg imports and for thrid party stuff

**Comment by cdrage** [comment]
Hey @kadel all of these newlines and such are automatically from gofmt / goimports. If I change it manually it'll just revert back to what it was before.

File: .travis.yml [hunk 3]
```diff
@@ -13,7 +13,7 @@ jobs:
     - &base-test
       stage: test
       go_import_path: github.com/redhat-developer/odo
-      go: "1.11"
+      go: "1.11.2"
```
Comment (mik-dass) [inline .travis.yml]: We need to also update the go version in deploy stage here https://github.com/redhat-developer/odo/blob/5622eb92ba3ece4d42ddd8c4bb00cf6f0b8c8ef0/.travis.yml#L109

**Pull request closed without merging**